### PR TITLE
feat: implement report problem feature, revert pn change

### PR DESCRIPTION
**Pull request by bryce-mcmath**
Leaving in draft until Bifold package deps are ready and updated but it's here if you want to see how the Bifold changes are used. Depends on:
- https://github.com/openwallet-foundation/bifold-wallet/pull/1359

### CODE REVIEW

File: app/container-imp.ts [hunk 1]
```diff
@@ -113,7 +114,6 @@ export class AppContainer implements Container {
       lokiUrl: Config.REMOTE_LOGGING_URL,
       lokiLabels: {
         application: getApplicationName().toLowerCase(),
-        job: 'react-native-logs',
```
Comment (bryce-mcmath) [inline app/container-imp.ts:116]: `job` is now specified by the remote logger, if we also specify it here, every job will be the same whether they are incident reports or remote logging sessions. Having separate jobs should allow us to have different lifespans for each job type

File: app/src/hooks/initialize-agent.ts [hunk 2]
```diff
@@ -190,7 +190,7 @@ const useInitializeBCAgent = () => {
     
     if (store.preferences.usePushNotifications) {
       logger.info('Activating push notifications...')
-      await activate(newAgent)
+      activate(newAgent)
```
Comment (bryce-mcmath) [inline app/src/hooks/initialize-agent.ts:193]: I'm hoping this reverted line will fix the issue Shel mentioned that happens during SauceLabs app initialization

**Review by bryce-mcmath — COMMENTED** [review]
Pre-review to clarify a couple things

**Review by al-rosenthal — APPROVED** [review]
🌮

**Review by jleach — APPROVED** [review]

**Pull request merged**